homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -3,16 +3,23 @@
 
 import voluptuous as vol
 
+import datetime as dt
+
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import (
-    CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME, SUN_EVENT_SUNSET)
+    CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME, SUN_EVENT_SUNSET, CONF_SCAN_INTERVAL)
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.sun import get_astral_event_date
 import homeassistant.util.dt as dt_util
+from homeassistant.helpers.event import async_track_sunset, async_call_later, async_track_time_change
+
+import hdate
 
 _LOGGER = logging.getLogger(__name__)
 
+SCAN_INTERVAL = dt.timedelta(days=9999)
+
```

Comment:
Not sure this has any effect here

Reply:
This prevents the recalculating every 30 seconds (the default)

Reply:
Ok, so maybe set it to a sane value say 1 day.

Reply:
Just wanted to disable it, but once a day is fine